@remirror/extension-tables/src/__tests__/table.spec.tsx

```diff
@@ -0,0 +1,201 @@
+/** @jsx jsx */
+
+import { jsx } from '@emotion/core';
+import { fromHTML, toHTML } from '@remirror/core';
+import { BaseKeymapExtension } from '@remirror/core-extensions';
+import { createBaseTestManager } from '@remirror/test-fixtures';
+import { shallow } from 'enzyme';
+import { pmBuild } from 'jest-prosemirror';
+import { renderEditor } from 'jest-remirror';
+
+import { TableCellExtension, TableExtension, TableRowExtension } from '..';
+
+describe('schema', () => {
+  const { schema } = createBaseTestManager([
+    { extension: new TableExtension() },
+    { extension: new TableRowExtension() },
+    { extension: new TableCellExtension() },
+  ]);
+
+  const { doc, table, tableRow, tableCell } = pmBuild(schema, {
+    table: { nodeType: 'table' },
+    tableRow: { nodeType: 'tableRow' },
+    tableCell: { nodeType: 'tableCell' },
+  });
+
+  const node = table(
+    tableRow(tableCell('A1'), tableCell('B1')),
+    tableRow(tableCell('A2'), tableCell('B2')),
+    tableRow(tableCell('A3'), tableCell('B3')),
+  );
+
+  const html: string = shallow(
```

Comment:
This can be tested without enzyme. I think I'm using strings in other places in the codebase to test the same thing.

Reply:
An example from another test is shown below.
 https://github.com/remirror/remirror/blob/026d42385d912853718863134b7bfc2e751a008f/%40remirror/core-extensions/src/marks/__tests__/link-extension.spec.ts#L23-L27